do-not-remove/projects_guidelines.md

```diff
@@ -0,0 +1,48 @@
+﻿# Guidelines for our GitHub projects
+
+In this place we will have the documentation for how to handle issues
+among projects of **TecnoYucas**.
+
+## General notes
+
+### Some rules first about the project
+
+1. Every repo must have a automatic Kanban-like GitHub project
+2. Every single pull request MUST be associated with a GitHub issue
+3. Whenever you close a pull request or comment on an issue, please
+make sure to agree on issue completion.
+4. All issues should have a deliverable product (a document or a
+merged branch, this would help on issue completion agreement).
+
+### Some recommendations
+
+- Keep language as complete and concise as possible.
+- Use Todo lists (this can help to better define the scope of an issue
+  and the state of it).
+- Please try not to create huge issues, make them as short and clear
+  scoped as possible. If you need some sort of complex issue, please
+  break it down into smaller pieces.
+
+### The workflow
+
+1. When an issue is newly created, it will directly go into `Inbox
+(Not approved yet)` column
+2. After an issue is approved (by at least **two** members of the
```

Comment:
we can automate this if we use waffle instead of github projects and treat the boards to use labels, so an issue can go from Inbox to `To  do (Approved)` as soon as someone assigns him/herself to the issue.

Reply:
Cool, I can investigate and set a project in Waffle, what do you think @sadasant ?

Reply:
I haven't used Waffle, but Github Projects have something similar: https://help.github.com/articles/configuring-automation-for-project-boards/

Reply:
Yes! Well, essentially: all of the columns should be automated based on tags, once we do that, I'll be comfortable with having rules around this. None of this process should be manual (except for some of the issue assignment at least)